javascript/javascript3/week1/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Noto+Sans+Display:ital,wght@0,400;0,600;1,300&display=swap" rel="stylesheet">
+    
+</head>
+<body>
+  <div class="top-button">
+    <button id="location">Get Your Weather</button>
+  </div>
+    
+
+
+<div id="container">
+
+  <div class="top-container">
+    <h1 id= "heading"><h1>
+    <p id="temppara"></p>
+      <p id="feelslike"></p>
+    <img id="image" src= "">
+  </div>
+  
+    <div class="bottom-container">
+      <div class="sections">
+        <h4>Wind Speed</h4>
+        <p id="cl-1"></p>
```

Comment:
Never use some namings like "cl-1". Try to find something unique for that particular thing, more meaningful so that when you yourself or someone else is seeing your classname, would make sense.